config/prow/cluster/monitoring/mixins/lib/config_util.libsonnet

```diff
@@ -39,6 +39,6 @@
           ),
         },
     }
-    + default(config, 'prowImageStaleByDays', 7)
+    + default(config, 'prowImageStaleByDays', {daysStale: 7, hoursToTrigger: '24h'})
```

Comment:
nit: This won't default the fields individually, only the entire struct if it is omitted.

Reply:
Make sense, done